### ISSUE TITLE
Unclickable header links when search is focused

### ISSUE THREAD

**Issue by DannyBen** [retypeapp/retype#345]
In smaller screen width, when focusing the search bar, it would seem like an invisible overlay is preventing clicks on the top navigation links.

In the below screencast, the Discussions and Twitter links cannot be clicked.
![out](https://user-images.githubusercontent.com/2405099/175813795-10d595b1-e3a6-4c57-a754-54423359246d.gif)

I guess it is an over-zealous `div` overaly.

**Comment by geoffreymcgill** [comment]
Good catch. Thanks for letting us know about this issue. We'll try and fix right away.
